### ISSUE TITLE
iOS Accessibility - VoiceOver rotor functions are unsupported.

### ISSUE THREAD

**Issue by SergeantConfused** [bitwarden/mobile#2589]
### Steps To Reproduce

1. Activate VoiceOver on iOS.
2. Open Bitwarden and create a new vault item.
3. Use Rotor Functions (https://support.apple.com/en-us/HT204783) set to Words.

### Expected Result

VoiceOver should navigate through the words in the form.

### Actual Result

VoiceOver does not read any words from the form.

### Screenshots or Videos

N/A.

### Additional Context

It also does not work when Rotor functions are set to (Characters).

### Operating System

iOS

### Operating System Version

16.5.1

### Device

iPhone SE (2020)

### Build Version

2023.5.0

### Beta

- [ ] Using a pre-release version of the application.

**Comment by closebot-bw** [comment]
This repo has been archived. To report an issue you're experiencing in the new native apps, visit the [iOS](https://github.com/bitwarden/ios) / [Android](https://github.com/bitwarden/android) repo.
